RNChat/README.md

```diff
@@ -1,44 +1,140 @@
+Clone the project:
 
-Prepare environment for React Native and:
+```bash
+# choose a folder on your computer to clone the "connectycube-reactnative-samples" repo
+cd path/to/folder
+# use git to clone
+git clone https://github.com/ConnectyCube/connectycube-reactnative-samples.git
+# navigate to React Native Chat sample project
+cd connectycube-reactnative-samples/RNChat
+```
 
-1. Clone the project;
-2. Install node_modules: `cd connectycube-js-samples/RNChat && npm install`
-3. In order to use push notifications on Android, you need to create `google-services.json` file and copy it into project's `android/app` folder. Also, you need to update the `applicationId` in `android/app/build.gradle` to the one which is specified in `google-services.json`, so they must match. If you have no existing API project yet, the easiest way to go about in creating one is using this [step-by-step installation process](https://firebase.google.com/docs/android/setup)
-4. In order for push notifications to work properly - it requires to do create a key/certificate and upload to ConnectyCube admin panel. The complete guide is available here https://developers.connectycube.com/reactnative/push-notifications
-5. Run `npm run ios` or `npm run android`.
+### Step 2: Install _node_modules_
+
+From RNChat project root:
+
+```bash
+yarn install
+```
+
+### Step 3: Start the Metro Server
+
+First, you will need to start **Metro**, the JavaScript _bundler_ that ships _with_ React Native.
+To start Metro, run the following command from the _root_ of your React Native project:
+
+```bash
+# using Yarn
+yarn start
+```
+
+### Step 4: Start your Application
+
+Let Metro Bundler run in its _own_ terminal. Open a _new_ terminal from the _root_ of your the RNChat project. Run the following command to start your _Android_ or _iOS_ app:
+
+### For Android
+
+```bash
+# using Yarn
+yarn android
+```
+
+### For iOS
+
+Open _RNChat/ios/RNChat.xcworkspace_ in Xcode and select iOS device target or:
+
+```bash
+# using Yarn
+yarn ios
+```
+
+## Configure Push Notification
+
+ConnectyCube React Native Push Notification [guide](https://developers.connectycube.com/reactnative/push-notifications)
+
+## Running on device
+
+This project use React Native Notifications and it's recommended to test your app on an actual physical [Android](https://reactnative.dev/docs/running-on-device?platform=android)/[iOS](https://reactnative.dev/docs/running-on-device?platform=ios) devices.
+
+## Build your own chat application
+
+To make the sample works for your own app, please do the following:
+
+1. Register new account and application at [https://admin.connectycube.com](https://admin.connectycube.co)
+
+2. Put Application credentials from 'Overview' page into `connectycube_config.json` file:
+
+```json
+{
+  "connectyCubeConfig": [
+    {
+      "appId": 0,
+      "authKey": "***",
+      "authSecret": "***"
+    },
+    {
+      ...
+      "chat": {
```

Comment:
let's make it simper for users. Let's remove all these custom settings and keep credentials only